users/engler/fairseq/training.py

```diff
@@ -0,0 +1,97 @@
+import os
+import subprocess as sp
+import yaml
+
+from sisyphus import *
+
+import recipe.i6_core.util as util
+
+
+class DictToYamlJob(Job):
+    """
+    Writes a dict into a .yaml file
+    """
+
+    def __init__(self, config_dict, hash_reset, *, yaml_prefix=""):
+        """
+        :param dict config_dict:
```

Comment:
The job name is generic and not related to fairseq. However, this variable implies that what is written is always a config. I'd either make this a `FairseqConfig` or `FairseqYamlConfig` with a `write` method which is called in a `create_files` task of the `FairseqHydraTrainingJob` similar to the `ReturnnConfig` or leave this job generic.